src/backend/expungeservice/models/record_summary.py

```diff
@@ -17,7 +17,7 @@ class RecordSummary:
     record: Record
     total_charges: int
     cases_sorted: Dict[str, List[str]]
-    eligible_charges: List[str]
+    eligible_charges_by_date: List[List[Any]] # Each list element is a list of length 2: [str, List[str]]
```

Comment:
Do you know about `Tuple[str, List[str]]`?

Reply:
Ah cool. I did not